__tests__/app.test.js

```diff
@@ -351,4 +351,23 @@ describe("POST /api/articles/:article_id/comments", () => {
             expect(body.msg).toBe("The username is not found");
           });
       });
+  });
+
+  describe("DELETE /api/comments/:comment_id", () => {
+    test("GET 204: deletes comment with passed the comment_id then returns no content", () => {
+      return request(app)
+        .delete("/api/comments/6")
+        .expect(204)
+        .then(({ body }) => {
```

Comment:
A delete will not have a value, so we don't need to check the body here :) What you can do is you can make a query for this specific comment_id and assert that it's not in the database anymore.